janusgraph-codepipelines-ci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM maven:3-jdk-8
+ENV DEBIAN_FRONTEND noninteractive
+COPY settings-docker.xml /usr/share/maven/ref/
+ENV COMMIT 9165ee8407b3607f39031cdaeff1d15ba0fd4e9e
```

Comment:
It is a recent commit I can bake into a Docker repository so that building code does not take a long time - I use it to pull all the maven dependencies. Any better ideas?